README.md

```diff
@@ -26,69 +26,70 @@ Install the Docker generator:
 npm install -g generator-docker
 ```
 
-Run the generator in the same folder that your project is in:
+## Getting Started
+Once yo docker and VS Code are installed, here's the few steps to debug your first app
+### Node.js
+- From the root of your project, where your server.js file existts, open a terminal window
```

Comment:
That is not the entrypoint that is scaffolded, it uses nodemon or npm start both of which leverage project.json to find the correct fiel to start.

---
In reply to: [68331724](https://github.com/Microsoft/generator-docker/pull/90#discussion_r68331724) [](ancestors = 68331724)